src/web/src/main.js

```diff
@@ -13,10 +14,24 @@ Vue.config.productionTip = false;
 
 Vue.use(VueRouter);
 Vue.use(VueCookies);
+Vue.use(Vuex);
+
+const store = new Vuex.Store({
```

Comment:
I'd suggest moving the `store` declaration to its own file, then import store here.
Can put it in `src/web/src/store/index.js` then you do `import store from "../store"` or something along those lines